src/components/VHeader/VSearchBar/VSearchBar.vue

```diff
@@ -12,6 +12,7 @@
       type="search"
       name="q"
       :model-value="searchText"
+      v-on="{ blur: $listeners.blur, focus: $listeners.focus }"
```

Comment:
We don't want to pass _all_ listeners here because it breaks the input behavior, so I'm only passing the ones I need.